### ISSUE TITLE
deprecated api problem

### ISSUE THREAD

**Issue by shuvam-koirala** [Baseflow/flutter-permission-handler#428]
### Reproduction steps
use permission-handler package in your flutter project 
build the apk file in release mode

**Version:**^5.0.1+1

**Platform:** 
- [ ] :iphone: iOS
- [yes] :robot: Android

**Comment by ekremkmz** [comment]
Same error here
Note: C:\src\flutter\.pub-cache\hosted\pub.dartlang.org\permission_handler-5.0.1+1\android\src\main\java\com\baseflow\permissionhandler\PermissionHandlerPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

**Comment by jordantsap** [comment]
Same to me too

**Comment by lukepighetti** [comment]
```
Note: /Users/lukepighetti/.pub-cache/hosted/pub.dartlang.org/permission_handler-5.0.1+1/android/src/main/java/com/baseflow/permissionhandler/PermissionHandlerPlugin.java uses or overrides a deprecated API.
```

**Comment by jordantsap** [comment]
I think it's flutter related, i suppose you all use master or dev channel?? because it does not do it in stable

**Comment by mvanbeusekom** [comment]
This is fixed in version 6.0.1 of the permission_handler.
